### PR TITLE
Separate dockerfile for android ci

### PR DESCRIPTION
**Pull request by mniewrzal**
This is mainly reverted part with Android SDK + NDK installation from old version of main dockerfile. It can be minimized by removing unused elements but I leave the cleanup for later. For now, I would like to have something to run `uplink-android` tests.

### CODE REVIEW

File: android/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,124 @@
+FROM golang:1.14.4
```
Comment (stefanbenten) [inline android/Dockerfile:1]: Lets use 1.14.6 here

File: android/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+
+# CockroachDB
+
+RUN wget -qO- https://binaries.cockroachdb.com/cockroach-v20.1.1.linux-amd64.tgz | tar  xvz
+RUN cp -i cockroach-v20.1.1.linux-amd64/cockroach /usr/local/bin/
```
Comment (stefanbenten) [inline android/Dockerfile:6]: I think 20.1.3 is available :)

File: android/Dockerfile (hunk 1 above)
Comment (mniewrzal) [reply android/Dockerfile:1]: ok

File: android/Dockerfile (hunk 2 above)
Comment (mniewrzal) [reply android/Dockerfile:6]: this part can be removed for now. Android integration tests are using only postgres currently.

**Pull request merged**